README.md

```diff
@@ -1,2 +1,16 @@
-# hw01
-Repository for first CFSS homework assignment
+---
+title: "A Brief Introdution"
+author: "Eleanor Koh"
+date: "11/01/2017"
+output: html_document
+---
+I like elephants, coffee and *Barefoot Contessa*. 
+
+**3 fun facts**
+
+1. Apart from myself, elephants are the only mammals that can't jump. That's why we get along, elephants and I. Elephants are also remarkably sentient: they can laugh, cry and *hug* by wrapping their trunks together. 
+
+2. I own an espresso pot, a french press, and a v60 (this is a funnel) and a jar of instant coffee. This means that I can make coffee *4* different ways. 
+
+3. You know those 'if you could have dinner with someone who would it be?' questions? I would answer with *Ina Garten*. She worked in the White House, and then owned a gourmet food store, and now she runs her own celebrity cooking show. You gotta respect someone who's done so many things! (*And, as is quite clear, she can cook a mean meal.*)
+Check out her website here: <https://barefootcontessa.com/>
```

Comment:
Agreed! Practice embedding images in your Markdown documents (either internal or external images)